rpcs3/rpcs3qt/main_window.cpp

```diff
@@ -507,6 +507,7 @@ void main_window::InstallPackages(QStringList file_paths)
 		}
 
 		file_paths.append(paths);
+		m_gui_settings->SetValue(gui::fd_install_pkg, file_paths[0].path());
```

Comment:
Probably needs:
`
const QFileInfo file_info(file_paths[0]);
m_gui_settings->SetValue(gui::fd_install_pkg, file_info.path());
`

Reply:
Forgot to even check it compiled, fixed :D.